rama-crypto/src/jose/constants.rs

```diff
@@ -0,0 +1,52 @@
+pub(crate) use der_encoding_tags::*;
+pub(crate) use rsa_algorithm_identifier::RSA_ALGORITHM_IDENTIFIER;
+mod der_encoding_tags {
+    /// Identifier tag for a DER encoded integer.
+    /// Defined in [ITU X.680](https://www.itu.int/rec/T-REC-X.690/).
+    pub(crate) const DER_TAG_INTEGER: u8 = 0x02;
+    /// Identifier tag for a DER encoded bit string.
+    /// Defined in [ITU X.680](https://www.itu.int/rec/T-REC-X.690/).
+    pub(crate) const DER_TAG_BIT_STRING: u8 = 0x03;
+    /// Identifier tag for a DER encoded sequence.
+    /// Defined in [ITU X.680](https://www.itu.int/rec/T-REC-X.690/).
+    pub(crate) const DER_TAG_SEQUENCE: u8 = 0x30;
+    /// Maximum length of a DER encoded length in short form.
+    /// Defined in [ITU X.690](https://www.itu.int/rec/T-REC-X.690/).
+    pub(crate) const DER_LENGTH_SHORT_FORM_MAX: usize = 127;
+    /// Octet that indicates that no unused bits are present in a bit string.
+    /// Defined in section 8.6 of [ITU X.690](https://www.itu.int/rec/T-REC-X.690/).
+    pub(crate) const BIT_STRING_NO_UNUSED_BITS: u8 = 0x00;
+}
+
+/// Byte representation of RSA encryption algorithm identifier.
+/// See appendix C of [RFC 8017](https://datatracker.ietf.org/doc/rfc8017/)
```

Comment:
Seems like an odd resource to link. Yes some if it is vaguely appears in these RFCs... but not all and even the ones that are are just by sheer luck of an example... Is there no authorive table for these???? I feel like I have to be sherlock holmes to try to find it. How do you find this info while implementing it????